### PR TITLE
Remove line length violations for o.e.cluster (mostly)

### PR DESCRIPTION
**Pull request by rjernst**
This commit removes line length violations in most of the classes under
org.elasticsearch.cluster.

### CODE REVIEW

**Comment by rjernst** [comment]
This PR was getting unwieldy so I capped it around 1k lines changed. I will do the rest of o.e.cluster in a followup.

File: server/src/main/java/org/elasticsearch/cluster/ClusterStateUpdateTask.java [hunk 1]
```diff
@@ -28,7 +28,8 @@
 /**
  * A task that can update the cluster state.
  */
-public abstract class ClusterStateUpdateTask implements ClusterStateTaskConfig, ClusterStateTaskExecutor<ClusterStateUpdateTask>, ClusterStateTaskListener {
+public abstract class ClusterStateUpdateTask
+    implements ClusterStateTaskConfig, ClusterStateTaskExecutor<ClusterStateUpdateTask>, ClusterStateTaskListener {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/cluster/ClusterStateUpdateTask.java]: Could you indent this one extra time?

File: server/src/main/java/org/elasticsearch/cluster/ClusterStateUpdateTask.java [hunk 2]
```diff
@@ -41,7 +42,8 @@ public ClusterStateUpdateTask(Priority priority) {
     }
 
     @Override
-    public final ClusterTasksResult<ClusterStateUpdateTask> execute(ClusterState currentState, List<ClusterStateUpdateTask> tasks) throws Exception {
+    public final ClusterTasksResult<ClusterStateUpdateTask> execute(ClusterState currentState, List<ClusterStateUpdateTask> tasks)
+        throws Exception {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/cluster/ClusterStateUpdateTask.java]: Could you indent this one extra time?

**Review by nik9000 — APPROVED** [review]
Left a minor thing but LGTM regardless.

**Comment by rjernst** [comment]
This PR is purely mechanical and the CI failure is transient (does not reproduce). I'm going to push this as is.

**Pull request merged**